ui: fix mixed units in tooltips

vladlos opened the pull request:
We already calculate biggest scale for chart axis label, so I take that and cast values in chart tooltips to fit chart scale. So tooltips will show consistent units, matching the biggest unit type in data set and axis label. And also has 2 numbers after dot for better readability. This was done for duration and bytes units.

Resolves: #35467

Release note (ui): consistent units in chart tooltips

result:
![Screen Shot 2019-09-23 at 6 11 50 PM](https://user-images.githubusercontent.com/12850886/65438206-c2868080-de2d-11e9-9058-be07dee2c86a.png)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/cockroachdb/cockroach?pullRequest=40970) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/cockroachdb/cockroach?pullRequest=40970) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/cockroachdb/cockroach?pullRequest=40970) it.</sub>

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/40970)
<!-- Reviewable:end -->

nathanstilwell reviewed (commented):
The code looks good, but I would like to see some unit tests for the format functions. This made me realize that none of the other format utils are tested, but I suppose we have to start somewhere.

nathanstilwell commented:
@elkmaster You may disregard the this build failure for now. When I release branch is cut the _justification_ requirement will be lifted.

vladlos committed:
ui: fix mixed units in tooltips

We already calculate biggest scale for chart axis label, so I take that and cast values in chart tooltips to fit chart scale. So tooltips will show consistent units, matching the biggest unit type in data set and axis label. And also has 2 numbers after dot for better readability. This was done for duration and bytes units.

Resolves: #35467

Release note (ui): consistent units in chart tooltips

elkmaster commented:
@nathanstilwell added unit tests for my code
